<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAvatarDlg</class>
 <widget class="QWidget" name="CreateAvatarDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Image:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KUrlRequester" name="urImage"/>
   </item>
   <item row="2" column="1">
    <widget class="KImageFilePreview" name="ifImage"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KImageFilePreview</class>
   <extends>QWidget</extends>
   <header>kimagefilepreview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>urImage</sender>
   <signal>urlSelected(KUrl)</signal>
   <receiver>ifImage</receiver>
   <slot>showPreview(KUrl)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
